packages/element/src/token-count.js

```diff
@@ -0,0 +1,25 @@
+let tokenCount = {};
```

Comment:
Ideally, this should be a part of `createInterpolateElement` - so you don't have to call `resetTokenCount` each time but rather treat it as an internal state.

Reply:
Ya this is another extraction artifact, I'll move into the main file.

Reply:
handled in 8711d13